packages/web/src/components/file-replace-container/FileReplaceContainer.tsx

```diff
@@ -33,20 +34,28 @@ export const FileReplaceContainer = ({
   return (
     <Flex justifyContent='space-between' alignItems='center' gap='l'>
       <PlainButton
+        size='large'
         iconLeft={isPlaying ? IconPause : IconPlay}
         onClick={onTogglePlay}
       >
         {fileName}
       </PlainButton>
       <Flex gap='s'>
-        <Button
-          variant='secondary'
-          size='small'
-          iconLeft={IconArrowUpToLine}
-          onClick={onClickReplace}
+        <ReactDropzone
+          // empty style makes this hug the button
+          style={{}}
+          multiple={false}
+          accept='audio/*'
```

Comment:
I noticed this accepted mp4 in the file picker, but it didn't make it through `processFiles` even though it's an accepted format. Maybe worth checking or trying to repro. Could just be my file was bad.